stock_picking_procure_method/readme/CONFIGURE.rst

```diff
@@ -11,7 +11,7 @@ Common steps to `mrp` and `purchase` procurements:
 #. Create **WH2** setting off *Manufacture to Resupply* and *Buy to
    Resupply*. Set **WH1** as the *Resupply Warehouse*.
 #. Go to *Inventory > Configuration > Warehouse Management > Routes* and click
-   on the *Make To Order* one.
+   on the *Manufacture* one.
```

Comment:
This one should be the same. The difference is that now such route is archived by default.